lib/mentor/home.dart

```diff
@@ -1,90 +1,90 @@
+    Widget build(BuildContext context) {
+      return Scaffold(
+      backgroundColor: Colors.white,
+          body: Stack(
+              children: [
+                Align(
+                  alignment: Alignment.bottomCenter,
+                  child: new ButtonTheme.bar(
+                    child: new ButtonBar(
+                      alignment: MainAxisAlignment.center,
+                      children: <Widget>[
+                        new RaisedButton(
+                          onPressed: tmpFunction, //Should direct to My Account Screen
+                          color: Color(0xff808786),
+                          textColor: Colors.white,
+                          splashColor: Colors.grey,
+                          child: Text(" My Account "),
+                        ),
+                        new RaisedButton(
+                          onPressed: () {
+                            Navigator.pop(context);
+                          },
+                          color: Color(0xff808786),
+                          textColor: Colors.white,
+                          splashColor: Colors.grey,
+                          child: Text('Log Out'),
+                        ),
+                      ],
+                    ),
+                  ),
+                ),
+                Center
+                  (
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.center,
+                    children:[
+                      Text("Welcome back, *Mentor*",
+                        textScaleFactor: 2,
+                        textAlign: TextAlign.justify,
+                      ),
+                      Text("These are your tools!",
+                        textScaleFactor: 1,
+                        textAlign: TextAlign.justify,
+                      ),
+
+                      RaisedButton(
+                        child: Text(" Live Chat with Mentee "),
+                        onPressed: tmpFunction,
+                        color: Color(0xff33D2B0),
+                        textColor: Colors.white,
+                        splashColor: Colors.grey,
+                        padding: EdgeInsets.fromLTRB(10, 10, 10, 10),
+                      ),
+
+                      RaisedButton(
+                        child: Text(" Schedule "),
+                        onPressed: tmpFunction,
+                        color: Color(0xff22CDF7),
+                        textColor: Colors.white,
+                        splashColor: Colors.grey,
+                        padding: EdgeInsets.fromLTRB(10, 10, 10, 10),
+                      ),
+
+                      RaisedButton(
+                        child: Text(" List of Mentees "),
+                        onPressed: tmpFunction,
+                        color: Color(0xff096BD8),
+                        textColor: Colors.white,
+                        splashColor: Colors.grey,
+                        padding: EdgeInsets.fromLTRB(10, 10, 10, 10),
```

Comment:
These buttons seem mostly copy pasted, right? Can we pull them out into a function?

Reply:
I didnt touch Mentor screen at all, just left as placeholders until Prep tells us what they wanted the mentor screen to look like

Reply:
Ok, sure. In that case I think just remove the .DS_Store file and add that to the .gitignore and we can go ahead and merge this